assets.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This is not in the right place.   This shouldn't be hanging out in the repro root.

Reply:
Woops, lol thanks for reviewing, I've been testing and accidentally committed this.